libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -2074,13 +2157,17 @@ MQTTStatus_t MQTT_GetIncomingPacketTypeAndLength( MQTTTransportRecvFunc_t readFu
     }
     else if( ( status != MQTTBadParameter ) && ( bytesReceived == 0 ) )
     {
+        LogError( ( "No data was received from the transport." ) );
         status = MQTTNoDataAvailable;
     }
 
     /* If the input packet was valid, then any other number of bytes received is
      * a failure. */
     else if( status != MQTTBadParameter )
     {
+        LogError( ( "A single byte was not read from the transport: "
+                    "transportStatus=%d",
+                    status ) );
```

Comment:
There is a function to convert error status to Strings. It will be better to use that instead of the integer values of the status.

Reply:
ok thanks

Reply:
Sorry this should be "bytesReceived" as the printed thing